templates/example.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Dask (Python)",
```

Comment:
Since we're giving it a `title` distinct from `name`, could we also give it a like, `subtitle` distinct from `description`?

Reply:
Oh and `subtitle` would ideally be markdown so we could include links explicitly

Reply:
Yep!

Reply:
> Oh and `subtitle` would ideally be markdown so we could include links explicitly

Not yet. We can track that as a separate issue though.